gruntfile.js

```diff
@@ -345,7 +345,6 @@ module.exports = function (grunt) {
                     port: 9000,
                     middleware: function (connect) {
                         return [
-                            delayApiCalls,
```

Comment:
Delay is used in REST demo to allow the user to see the loading message. Should be kept.